### PR TITLE
Target: convert static members to fields

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]
LGTM, I'll let @ibuclaw double-check

**Review by jacob-carlborg — APPROVED** [review]

**Review by RazvanN7 — APPROVED** [review]

File: src/dmd/target.d [hunk 1]
```diff
@@ -95,17 +90,14 @@ struct Target
...
 
-    ///
-    alias FloatProperties = FPTypeProperties!float;
-    ///
-    alias DoubleProperties = FPTypeProperties!double;
-    ///
-    alias RealProperties = FPTypeProperties!real_t;
+    FPTypeProperties!float FloatProperties;     ///
+    FPTypeProperties!double DoubleProperties;   ///
+    FPTypeProperties!real_t RealProperties;     ///
```
Comment (Geod24) [inline src/dmd/target.d]: Why was a typedef/alias changed to a field ?

Comment (WalterBright) [reply src/dmd/target.d]: Because the `__gshared` was removed. See original line 71.

Comment (Geod24) [reply src/dmd/target.d]: Right, I got the nesting wrong. Thanks

**Pull request merged**